cfg.y

```diff
@@ -638,6 +641,23 @@ alias_def:	listen_id						{ IFOR();
 		 | phostport
 		 ;
 
+id_lst_param: WILDCARD_MATCH { IFOR();
```

Comment:
This creates a list of sockets from the alias and then checks whether the wildcard_match is appended as an option, similar to anycast on the actual socket definitions and adds it as a flag when creating the alias